tasks/66_sum-primes/sum-primes.js

```diff
@@ -0,0 +1,15 @@
+export function sumPrimes(num) {
+  let primeSum = 0;
+  for (let i = 2; i <= num; i++) {
+    if (isPrime(i)) {
+      primeSum += i;
+    }
+  }
+  return primeSum;
+}
+function isPrime(num) {
+  for (let i = 2; i <= Math.sqrt(num); i++) {
+    if (num % i === 0) return false;
```

Comment:
Please try to remain consistency and use `{ }` for if/else statement. 
```suggestion
    if (num % i === 0) {
      return false;
    }
```